src/resolve/resolved-manager.h

```diff
@@ -18,6 +18,7 @@ typedef struct Manager Manager;
 #include "resolved-dns-server.h"
 #include "resolved-dns-stream.h"
 #include "resolved-dns-trust-anchor.h"
+#include "resolved-dnstls.h"
```

Comment:
Please add `#if ENABLE_DNS_OVER_TLS`.

BTW, I think the `#if` condition should be moved to `resolved-dnstls.h`.

Reply:
I've moved the condition with commit 0b5c09b